docs/attributes-registry/gen-ai.md

```diff
@@ -28,6 +29,18 @@ This document defines the attributes used to describe telemetry in the context o
 **[1]:** It's RECOMMENDED to format completions as JSON string matching [OpenAI messages format](https://platform.openai.com/docs/guides/text-generation)
 **[2]:** It's RECOMMENDED to format prompts as JSON string matching [OpenAI messages format](https://platform.openai.com/docs/guides/text-generation)
 
+`gen_ai.operation.name` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value                  | Description          | Stability                                                        |
+| ---------------------- | -------------------- | ---------------------------------------------------------------- |
+| `completions`          | Completions          | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
Howa about adding `tokenize` as well?

tokenize(prompt, return_tokens=False)[[source]](https://ibm.github.io/watson-machine-learning-sdk/_modules/ibm_watson_machine_learning/foundation_models/model.html#Model.tokenize)
The text tokenize operation allows you to check the conversion of provided input to tokens for a given model. It splits text into words or sub-words, which then are converted to ids through a look-up table (vocabulary). Tokenization allows the model to have a reasonable vocabulary size.